packages/hash/frontend/src/components/layout/PageSidebar/AccountPageList/AccountPageList.tsx

```diff
@@ -16,7 +16,7 @@ import { useCreatePage } from "../../../hooks/useCreatePage";
 
 type AccountPageListProps = {
   accountId: string;
-  currentPageEntityId: string;
+  currentPageEntityId?: string;
```

Comment:
@teenoh I’ve made this parameter optional so that we can see the list of pages even if we browse a type. This behavior feels more logical to me, but I’m happy to change it back if the spec suggests otherwise.